TwitchPlaysAssembly/Src/ComponentSolvers/Modded/Misc/AbstractSequencesComponentSolver.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Collections;
+
+public class AbstractSequencesComponentSolver : ReflectionComponentSolver
+{
+	public AbstractSequencesComponentSolver(TwitchModule module) :
+		base(module, "abstractSequencesScript", "!{0} press <p1> (p2)... [Presses the numbered button(s) in the specified position(s)] | !{0} submit [Presses the submit button] | Valid positions are 1-16 in reading order")
+	{
+	}
+
+	public override IEnumerator Respond(string[] split, string command)
+	{
+		if (!command.Equals("submit") && !command.StartsWith("press ")) yield break;
+		if (command.Equals("submit"))
+		{
+			if (!_component.GetValue<bool>("canClickAgain")) yield break;
+
+			yield return null;
+			yield return "solve";
+			yield return "strike";
+			yield return Click(16, 0);
+		}
+		else if (command.StartsWith("press "))
+		{
+			for (int i = 1; i < split.Length; i++)
+			{
+				if (!split[i].EqualsAny("1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15", "16")) yield break;
```

Comment:
Perhaps it might be better to try to parse them into a number and then use `.InRange(1, 16)`.